src/Dotnet.Script/Program.cs

```diff
@@ -97,10 +97,11 @@ private static int Wain(string[] args)
             {
                 c.Description = "Creates a sample script along with the launch.json file needed to launch and debug the script.";
                 var fileName = c.Argument("filename", "(Optional) The name of the sample script file to be created during initialization. Defaults to 'main.csx'");
+                var cwd = c.Option("-cwd |--workingdirectory <currentworkingdirectory>", "Working directory for initialization. Defaults to current directory.", CommandOptionType.SingleValue);
                 c.OnExecute(() =>
                 {
-                    var scaffolder = new Scaffolder();
-                    scaffolder.InitializerFolder(fileName.Value);
+                    var scaffolder = new Scaffolder(new ScriptLogger(ScriptConsole.Default.Error, debugMode.HasValue()));
```

Comment:
why are we writing to error output here?

Reply:
I figured since we already do all logging to `stderr` we might just do it here as well.


https://github.com/filipw/dotnet-script/blob/c0b7cdf2f929eb539cd6d6ec3a89a3815e6ce5de/src/Dotnet.Script/Program.cs#L222-L224